src/mixins/validatable-http-message.coffee

```diff
@@ -14,11 +16,11 @@ class Validatable
   #@return [Object] :headers {ValidationErrors}, :body {ValidationErrors}, :statusCode [Boolean]
   validate: () ->
     @validation = {}
-
+    @validation.version = '2'
     @lowercaseHeaders()
 
-    @validateHeaders() unless @headers == undefined
-    @validateBody() unless @body == undefined
+    @validateHeaders() unless @headers == undefined || @expected.headers == undefined
```

Comment:
Why this change? It just brings another type of behavior. Do we really want "not to validate headers" if one of real/expected `headers` is missing?

dtto for `body`